ckanext/datastore/plugin.py

```diff
@@ -282,21 +282,9 @@ def before_show(self, resource_dict):
                 controller='ckanext.datastore.controller:DatastoreController',
                 action='dump', resource_id=resource_dict['id'])
 
-        connection = None
+        if 'datastore_active' not in resource_dict:
+            resource_dict[u'datastore_active'] = False
```

Comment:
I seem to recall this is just a catch all for when datastore_active isn't set at all on the resource for some reason. It can probably go as long as we definitely set datastore_active: False on all existing resources.